library/src/scala/quoted/show/package.scala

```diff
@@ -0,0 +1,7 @@
+package scala.quoted
+
+package object show {
+
+  def apply[T](expr: given QuoteContext => Expr[T]) given (toolbox: Toolbox): String = run(expr.show.toExpr)
+
+}
```

Comment:
Why not just `expr.show` instead of `run(expr.show.toExpr)`?

Reply:
Because the prefix is of type `Expr[T]` but we need a `given QuoteContext => Expr[T]` which we cannot encode as an extension method

Reply:
Aslo, this `show` intanciated a new compiler run which we should never do inside a run or a macro. Better to not get the two method confused

Reply:
It would be clearer for the users if we remove this variant of `show` and use only the `Expr.show`. The latter one can only be called within the a `run { ... }` or a macro `${ ... }`. In case they write tests and only care about showing the result of computing some closed expression they could write `run(expr.show.toExpr)` themselves.